src/SourceBuild/content/repo-projects/fsharp.proj

```diff
@@ -18,6 +18,10 @@
       custom sourcebuild arg is the one that's required. This avoids running bootstrapping twice.
     -->
     <BuildArgs Condition="'$(DotNetBuildSourceOnly)' == 'true'">$(BuildArgs) $(FlagParameterPrefix)sourcebuild $(FlagParameterPrefix)tfm $(NetCurrent) /p:SourceBuildBootstrapTfm=$(NetCurrent)</BuildArgs>
+        
+    <!-- See https://github.com/dotnet/source-build/issues/4603. 
+    Should be removed with https://github.com/dotnet/fsharp/issues/17713 -->
+    <NoWarn Condition="'$(DotNetBuildSourceOnly)' == 'true'">FS0064</NoWarn>
```

Comment:
Doesn't think need to flow into the inner FSharp build?  How does that work?  All of the infra to flow nowarns was removed.  I was expecting a patch was necessary to fix this.

Reply:
Oh interesting. It passed on my local build. I'll pull the binlogs from that and double check that this flows to the inner FSharp build.

Reply:
Ok yes, you were right, the `NoWarn` doesn't flow. I'm not sure why my local build passed, perhaps I pulled the wrong SDK or the error doesn't happen on ubuntu2404 stage 2 builds? Not sure.

Regardless, FSharp has a fix out in https://github.com/dotnet/fsharp/pull/17748 so I'll patch with that.